jaxrs-spec/src/main/asciidoc/chapters/environment/_javaee.adoc

```diff
@@ -190,6 +190,16 @@ Note that if JSON-B and JSON-P are both supported in the same
 environment, entity providers for JSON-B take precedence over those for
 JSON-P for all types except `JsonValue` and its sub-types.
 
+[[jaxb]]
+==== Jakarta XML Binding
+
+In a product that supports the Jakarta XML Binding API (JAXB)
+<<bib21>>, implementations MUST support entity providers for the type,
+`jakarta.xml.bind.JAXBElement`, and application-supplied JAXB classes
+in combination with the following XML-based media types:
+`text/xml` and `application/xml` and media types of the
+form `application/*+xml`.
```

Comment:
Is this syntax correct?  Or, should it just be `application/+xml`?

Reply:
It's a wildcard - it's for content types like `application/svg+xml` or `application/xslt+xml`.